easybuild/easyconfigs/s/Spyder/Spyder-4.1.5-foss-2020a-Python-3.8.2.eb

```diff
@@ -87,11 +87,11 @@ exts_list = [
     }),
     ('keyring', '21.4.0', {
         'checksums': ['9aeadd006a852b78f4b4ef7c7556c2774d2432bbef8ee538a3e9089ac8b11466'],
-        'preinstallopts': """sed -i 's/setup()/setup(version="%(version)s")/g' setup.py && """
+        # 'preinstallopts': """sed -i 's/setup()/setup(version="%(version)s")/g' setup.py && """
```

Comment:
Delete this line instead of commenting it out.

Reply:
Done